modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. From your centrally managed VPC, create or modify a VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*. 
+. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a new custom trust policy role for the shared VPC permissions. This role should have the following permissions:
+  * A trust policy to assume roles:
++
+[source,terminal]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+	  "Sid": "Statement1",
+	  "Effect": "Allow",
+	  "Principal": {
+	  	"AWS": "arn:aws:iam::<Account-ID>:root"
+	  }, <1>
+	  "Action": "sts:AssumeRole"
+	}
+  ]
+}
+----
++
+--
+<1> The below principals will be added later in this process once the *cluster-creating AWS account* user has created these roles. On creation, you should create a root user placeholder using the *cluster-creator's AWS Account* ID as `arn:aws:iam::{Account}:root`.
+--
+    * `ResourceGroupandTagEditorFullAccess` permissions policy
+    * `Route53minimalPermissions` permissions policy
++
+After you create this IAM role, provide the created role's ARN to the cluster creator.
+
+. In the link:https://us-east-1.console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a Resource Share that shares the previously created public and/or private subnets to the *cluster-creating AWS account* ID.
```

Comment:
I don't think we are supposed to use "and/or" because it causes translation errors.